### ISSUE TITLE
Movement_used and attacks_used keys should be represented in game

### ISSUE THREAD

**Issue by Hejnewar** [wesnoth/wesnoth#7367]
### Describe the desired feature

Currently two attack attributes in mainline exist without being represented anywhere. Movement_used and attacks_used keys. For better player experience as well as ease of use in mainline and UMC I believe that they should be shown somewhere within the UI.

**Comment by CelticMinstrel** [comment]
Suggestion for attacks used: "(3)" after the attack name (if attacks left and max attacks are shown elsewhere).

For movement used, maybe "(2 MP)". With both non-default, "(3; 2 MP)".

We definitely need to show attacks left and max attacks somewhere too.

**Comment by CelticMinstrel** [comment]
Ah, perfect. I missed that (it's probably only shown when actually relevant too).

**Comment by Wedge009** [comment]
If it's already present, does that mean the issue can be closed?

On the other hand, how did you 'activate' the extra information? I added attacks via debug mode and didn't see any difference. Presumably it's the unit definition that needs to be changed.

**Comment by Hejnewar** [comment]
> If it's already present, does that mean the issue can be closed?

No because the uses X attacks part is only in my mod and this was one of the two original problems. The attacks just show up if you put more than one max attack in a unit file but if the unit is reduced back to 1 attack they disappear so technically it should work with adding as well. 

 ``attacks = 2 ``

**Comment by CelticMinstrel** [comment]
> If it's already present, does that mean the issue can be closed?

No, this issue is about information on a specific attack – the number of movement points consumed when you use the attack. (And the amount it deducts from attacks_left.)

> On the other hand, how did you 'activate' the extra information?

For attacks_left and max_attacks I assume it's "activated" by having a max_attacks greater than 1.

For attacks_used I would expect it to be shown under the same condition, I think, but it could maybe be omitted if it's either equal to 1 or equal to max_attacks (pick one, not both).

For movement_used I would expect it to be shown whenever it's less than max_moves.
